src/main/scala/it/pagopa/interop/tenantprocess/api/adapters/ApiAdapters.scala

```diff
@@ -153,9 +154,12 @@ object ApiAdapters {
           DependencyTenantVerifier(
             id = requesterId,
             verificationDate = now,
-            renewal = seed.renewal.toDependency,
-            expirationDate = seed.expirationDate,
-            extensionDate = None
+            renewal = seed.renewal.map(_.toDependency).getOrElse(oldVerifier.renewal),
+            expirationDate = seed.expirationDate.orElse(oldVerifier.expirationDate),
```

Comment:
so far we have applied the "replace everything" logic, instead of the "replace only given fields" (because for example it is not otherwise possible to clear a field value).
I suggest to keep the current logic in order to avoid confusion of different behaviours for different endpoints.

```suggestion
            renewal = seed.renewal.map(_.toDependency),
            expirationDate = seed.expirationDate,
```

Reply:
I agree with @galales